tracer/tracer.go

```diff
@@ -105,10 +105,11 @@ type Tracer struct {
 	// when process events take place (new, exit, unknown PC).
 	triggerPIDProcessing chan bool
 
-	// pidEvents notifies the tracer of new PID events.
+	// pidEvents notifies the tracer of new PID events. Each PID event is a 64bit integer
+	// value, see bpf_get_current_pid_tgid for information on how the value is encoded.
 	// It needs to be buffered to avoid locking the writers and stacking up resources when we
 	// read new PIDs at startup or notified via eBPF.
-	pidEvents chan libpf.PID
+	pidEvents chan uint64
```

Comment:
Should a new type, maybe `libpf.PidTid`, be used here to make clear that these are not ordinary uint64 numbers?

Reply:
Added `libpf.PIDTID`